CRM/Civiquickbooks/Invoice.php

```diff
@@ -627,7 +708,7 @@ protected function mapToAccounts($db_contribution, $accountsID, $SyncToken, $qb_
       $new_invoice += array(
         'TxnDate' => $receive_date,
         'DueDate' => $due_date,
-        'DocNumber' => 'Civi-' . $db_contribution['id'],
+        'DocNumber' => $invoice_prefix . sprintf("%06d", $db_contribution['id']),
```

Comment:
Default invoice number will now look like: Civi-000001

I like my invoice numbers with leading zeros for better sortability. I can make this an option if you don't want it for everyone. Also now the prefix is easy to specify. (ours is CIV-)